Angular v14 not supported

### Describe the problem you'd like to have solved

When installing packages either through `npm i` or `ng update`, the process fails due to the `peerDependencies` of this package not accepting the newly released Angular Version 14.0.0.

### Describe the ideal solution

Amend the `peerDependencies` to include version 14, like was done in pull request #243 to support Angular 13.

## Alternatives and current work-arounds

The obvious alternative is to use the `--force` flag, but given the nature of this package's position in the app, for now I'll opt to remain on the previous version of Angular until this is resolved.

### Additional context: log of `ng update`

```⠸ Installing packages (npm)...npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: @auth0/auth0-angular@1.9.0
npm ERR! Found: @angular/common@14.0.0
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"^14.0.0" from the root project
npm ERR!   peer @angular/common@">=6.1.0" from @angular-slider/ngx-slider@2.0.3
npm ERR!   node_modules/@angular-slider/ngx-slider
npm ERR!     @angular-slider/ngx-slider@"^2.0.3" from the root project
npm ERR!   13 more (@angular/cdk, @angular/forms, @angular/material, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@">=9 <=13" from @auth0/auth0-angular@1.9.0
npm ERR! node_modules/@auth0/auth0-angular
npm ERR!   @auth0/auth0-angular@"^1.9.0" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: @angular/common@13.3.11
npm ERR! node_modules/@angular/common
npm ERR!   peer @angular/common@">=9 <=13" from @auth0/auth0-angular@1.9.0
npm ERR!   node_modules/@auth0/auth0-angular
npm ERR!     @auth0/auth0-angular@"^1.9.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

--- a/projects/auth0-angular/package.json
+++ b/projects/auth0-angular/package.json
@@ -26,9 +26,9 @@
   },
   "homepage": "https://github.com/auth0/auth0-angular#readme",
   "peerDependencies": {
-    "@angular/common": ">=9 <=13",
-    "@angular/core": ">=9 <=13",
-    "@angular/router": ">=9 <=13"
+    "@angular/common": ">=9 <=14",
+    "@angular/core": ">=9 <=14",
+    "@angular/router": ">=9 <=14"
   },
   "dependencies": {
     "tslib": "^2.0.0",

